### PR TITLE
Added OAuth sample

### PR DESCRIPTION
**Pull request by zachjonesnoel**
Added a sample that demonstrates Request Templates with OAuth.

### CODE REVIEW

File: request-template-with-oauth/config/requests.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "asanaGetWorkspace": {
+        "schema": {
+            "method": "GET",
+            "host": "app.asana.com",
+            "path": "/api/1.0/workspaces",
+            "headers": {
+                "Authorization": "bearer <%=access_token%>",
```
Comment (kaustavdm) [inline request-template-with-oauth/config/requests.json:8]: Can you change this for uniformity:

`"Bearer <%= access_token %>"`

**Review by kaustavdm — CHANGES_REQUESTED** [review]
Looks good overall. Can you make some cosmetic changes and then we are good to go? 

- See the comment on template literal usage.
- Run `npx prettier -w .` in the app directory.

File: request-template-with-oauth/config/requests.json (hunk 1 above)
Comment (zachjonesnoel) [reply request-template-with-oauth/config/requests.json]: Updated.

**Review by kaustavdm — APPROVED** [review]
Thanks!

**Pull request merged**